edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/version_2018_06_28/ModuleManagementHttpClient.cs

```diff
@@ -194,7 +200,8 @@ protected override void HandleException(Exception exception, string operation)
                     }
 
                 default:
-                    throw exception;
+                    ExceptionDispatchInfo.Capture(exception).Throw();
```

Comment:
There are a dozen or so more cases of ` throw (?!new)` under `edge-agent/` and `edge-hub/`. I guess we should fix those too?

Reply:
Sure, but this I want to merge to 1.0.7 and for that I want to keep the change smaller. I will send out a follow up with the other places fixed.

Reply:
Yeah, I didn't mean to fix them in *this* PR.